packages/fuselage/src/components/SideBarV2/SideBarItem/SideBarItem.tsx

```diff
@@ -0,0 +1,26 @@
+import type { AllHTMLAttributes } from 'react';
+import React from 'react';
+
+export const SideBarItem = ({
+  selected,
+  className,
+  children,
+  is: Tag = 'a',
+  ...props
+}: {
+  selected?: boolean;
+  is?: React.ElementType;
+} & AllHTMLAttributes<HTMLAnchorElement>) => (
```

Comment:
```suggestion
} & HTMLAttributes<HTMLAnchorElement>) => (
```

Reply:
`HTMLAttributes` doesn't include `href`, even with the generic being `HTMLAnchorElement` 😢